### DISCUSSION TITLE
Is it possible to make news widgets like in Windows 10 instead of the weather?

### DISCUSSION THREAD

**Discussion by vo1dpacket** [valinet/ExplorerPatcher#2435] [General]
Is this feature still in the Windows 11 regedit?

Instead of this:
![241907104-ff1ee90c-7737-40dc-b869-cc472c467210](https://github.com/valinet/ExplorerPatcher/assets/111048284/e730017c-612c-4910-9094-b6688987c6b5)

And this:
![241907359-2d073130-6841-43fb-a0b2-c601165f6aca](https://github.com/valinet/ExplorerPatcher/assets/111048284/ed646441-7f95-4583-9ba5-e66fbec14db6)

Replace with this menu:
![241907934-676a96e8-f69a-4bab-94ad-d2fe83e2b51e](https://github.com/valinet/ExplorerPatcher/assets/111048284/1384b2f9-640a-4e3f-b752-b8a09c426cb4)

**Comment by XuanChen1234** [comment]
You can ask from @Amrsatrio

**Comment by XuanChen1234** [comment]
@AkhilTehSpartan @HAYDENCRAFT2008 Any plans to make News and interests widgets like in Windows 10 instead of the weather?

**Reply by haydengriz** [reply]
not from what I know. I don't think it's doable

**Comment by XuanChen1234** [comment]
@valinet @Vipin2003 How do we add News widgets options in Windows 10, instead of the weather?
